docs/coreo/coreo.md

```diff
@@ -26,7 +26,7 @@ CloudCoreo CLI.
 * [coreo completion](coreo_completion.md)	 - Generate bash autocompletions script
 * [coreo configure](coreo_configure.md)	 - Configure CLI options
 * [coreo event](coreo_event.md)	 - Manage event stream
-* [coreo result](coreo_result.md)	 - Show the violation results
+* [coreo result](coreo_result.md)	 - Show the violation results(Deprecated, please follow the link to swagger API doc 'https://api.securestate.vmware.com')
```

Comment:
Those files are auto-generated by vss docs command. You can check https://github.com/spf13/cobra/blob/master/README.md#generating-documentation-for-your-command for more info. The code is under cli/cmd/doc.go